qrcode/console_scripts.py

```diff
@@ -67,6 +67,8 @@ def main(args=None):
 
     if args:
         data = args[0]
+        if sys.version_info[0] >= 3:
```

Comment:
The package is now Python3 only. Could you update the PR accordingly and explain why is this necessary?

Reply:
Fixed here: https://github.com/lincolnloop/python-qrcode/commit/1285c6d20feaaccee8f87959f6e7a17d41722dbe